packages/launch/src/focused-launch/index.tsx

```diff
@@ -23,12 +23,14 @@ function noop( ...args: unknown[] ) {
 }
 
 const FocusedLaunch: React.FunctionComponent = () => {
-	const { getSite } = useSelect( ( select ) => select( 'core' ) ) as any;
-	const title = getSite()?.title;
-
 	const site = useSite();
 
-	const [ siteTitle, setSiteTitle ] = React.useState( title );
+	const siteTitle = useSelect( ( select ) =>
+		select( SITE_STORE ).getSiteTitle( window._currentSiteId )
+	);
+
+	const { setSiteTitle } = useDispatch( SITE_STORE );
```

Comment:
What do you think about creating a custom hook so the interface could look like `useState`.
Example: `const [ siteTitle, setSiteTitle ] = React.useSiteTitle();`

Once everything is working fine here, this hook could be re-used in [step-by-step launch flow](https://github.com/Automattic/wp-calypso/blob/master/apps/editing-toolkit/editing-toolkit-plugin/editor-site-launch/src/launch-steps/name-step/index.tsx) or in some other integrations such as the preview-with-live-title https://cloudup.com/cRg0dZffLhu

Reply:
I can see that the `useTitle` hook has been updated in `packages/launch/src/hooks/use-title.ts`

@alshakero can you confirm that this addresses the [comment](https://github.com/Automattic/wp-calypso/pull/46921#discussion_r515000875) from @razvanpapadopol ?